config/params/config.go

```diff
@@ -316,6 +316,7 @@ type BeaconChainConfig struct {
 }
 
 type BlobScheduleEntry struct {
+	forkEpoch        bool
```

Comment:
this addition seems unrelated to the PR and isn’t mentioned in the description
I’m not opposed to it but it would be helpful to add some context
my guess is we want to use BlobScheduleEntry to distinguish between an actual fork and a BPO-only fork, adding some comments may make it clear

Reply:
woops! I will remove it. Yeah I am building another PR on top of this one for BPO handling.